### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by mcquenji**
- [x] Update the UI code to reflect the rename
- [x] Update README to reflect rename
  - [ ] Update screenshots with the new name

### CODE REVIEW

**Comment by mcquenji** [comment]
updating the screenshot on next release

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ChatGPT Citation Generator
+# ChatGPT Citations
 
 ## Overview
 The ChatGPT Citation Generator is a browser extension designed to generate BibTeX citations from chat conversations in OpenAI's ChatGPT interface. This tool aims to assist students, researchers, and anyone who needs to cite conversations or information obtained from ChatGPT in their academic or professional work.
```
Comment (officialbishowb) [inline README.md:4]: Maybe rename this to ChatGPT Citations also

Comment (mcquenji) [reply README.md]: done

**Review by officialbishowb — APPROVED** [review]
Looks good.

**Pull request merged**